### PR TITLE
// at line 1 only to make it different from master.

### PR DESCRIPTION
**Pull request by aerotekmusic**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,86 +52,86 @@ arm.flex();                             // 3)
...
-
-//  action 3:
-
+//  Noun: Mario
+//  action 1: Mario catches a coin
+mario.getCoin();
+//  action 2: Mario rides Yoshi
+mario.rideYoshi();
+//  action 3: Mario kisses Peach
+mario.kissPeach();
```
Comment (matkatmusic) [inline main.cpp:118]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

File: main.cpp [hunk 2]
```diff
@@ -52,86 +52,86 @@ arm.flex();                             // 3)
...
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: Neo
+//  action 1: neo fake awakes
+neo.fakeAwake();
+//  action 2: neo meets trinity
+neo.interactTrinity();
```
Comment (matkatmusic) [inline main.cpp:76]: `neo.meetTrinity();`
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -52,86 +52,86 @@ arm.flex();                             // 3)
...
-
-//  action 3:
-
+//  Noun: clock
+//  action 1: clock ticks
+clock.tick();
+//  action 2: clock alarm activates
+clock.alarmActivate();
+//  action 3: clock stops working
+clock.Stop();
```
Comment (matkatmusic) [inline main.cpp:86]: `clock.stop();`
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -52,86 +52,86 @@ arm.flex();                             // 3)
...
-
-//  action 3:
-
+//  Noun: Bass Player
+//  action 1: bass player practices bass 
+bassPlayer.practice();
+//  action 2: bass player gets good at bass
+bassPlayer.level1Up();
+//  action 3: bass player still doesn't get girls
+bassPlayer.chickRejection();
```
Comment (matkatmusic) [inline main.cpp:94]: `chickRejection` is not an action. 
`receiveRejectionFromChicks();` is how you would express this as an action
- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -52,86 +52,86 @@ arm.flex();                             // 3)
...
-
-//  action 3:
-
+//  Noun: Peter Pan
+//  action 1: peter pan flies
+peterpan.flies();
+//  action 2: peter pan fights captain hook
+peterpan.fightCapHook();
+//  action 3: peter pan smiles
+peterpan.smile();
```
Comment (matkatmusic) [inline main.cpp:102]: `peterPan`, not `peterpan`
- [ ] revise

File: main.cpp [hunk 6]
```diff
@@ -52,86 +52,86 @@ arm.flex();                             // 3)
...
-
-//  action 3:
-
+//  Noun: Michael Jackson
+//  action 1: michael jackson sings
+mj.sing();
+//  action 2: michael jackson dances
+mj.dance();
+//  action 3: michael jackson plays with kids
+mj.yesHeDid();
```
Comment (matkatmusic) [inline main.cpp:110]: `mj.playsWithKids();`
your pseudo-code needs to match the plain-English action almost identically.
- [ ] revise

File: main.cpp [hunk 7]
```diff
@@ -52,86 +52,86 @@ arm.flex();                             // 3)
...
-
-//  action 3:
-
+//  Noun: satellite
+//  action 1: satellite deploys solar panels
+satellite.deploySolarPanel();
+//  action 2: satellite relays signal
+satellite.relaySignal();
+//  action 3: satellite spies on russia
+satellite.spyRussia();
```
Comment (matkatmusic) [inline main.cpp:126]: `satellite.spyOnRussia();` 
Name functions what they do.  Don't omit words, as that reduces the readability of the code
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (aerotekmusic) [reply main.cpp:76]: done!

File: main.cpp (hunk 4 above)
Comment (aerotekmusic) [reply main.cpp:94]: ah, I understand the correct phrasing now. Thanks!

**Pull request merged**